### PR TITLE
Added architecture database

### LINKED ISSUE #3: Prepare diagram
- [x] ER diagram
- [x] Image
- [x] Draw file

### PR DESCRIPTION
**Pull request by MykytaKyt**
Prepare diagram

### CODE REVIEW

**Comment by MykytaKyt** [comment]
Danke

File: docs/diagram.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Description of database structure
+
+![image2.png](images/diagram.png)
```
Comment (aartiukh) [inline docs/diagram.md]: Please increase font size

File: docs/diagram.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table software consist of:
+
+var name | type    | constraints | description
+-------- | ------- | ----------- | ----------------------
+**id**   | PK, INT | NOT NULL    | stores id of software
+**name** | TEXT    | -           | store name of software
```
Comment (aartiukh) [inline docs/diagram.md]: constrains: not empty

File: docs/diagram.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table software_license consist of:
+
+var name             | type    | constraints | description
+-------------------- | ------- | ----------- | ------------------------------------------
+**id**               | PK, INT | NOT NULL    | stores id of a software license
+**software_id**      | FK, INT | NOT NULL    | refers to the table software
+**key_product**      | TEXT    | -           | stores product key of software
```
Comment (aartiukh) [inline docs/diagram.md]: Not empty?
Rename `key_product` to `product_key` (don't forget to update on the diagram image)

File: docs/diagram.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+**date_of_purchase** | DATE    | -           | stores info when the license was purchased
+**expires_date**     | DATE    | -           | stores the expiration date of the license
```
Comment (aartiukh) [inline docs/diagram.md]: constrains: `>= current date` or  `>= zero time` (1970)
Rename `expires_date` -> `date_of_expiry`

File: docs/diagram.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table employee_sw_license consist of:
+
+var name                | type    | constraints | description
+----------------------- | ------- | ----------- | --------------------------------------
+**id**                  | PK, INT | NOT NULL    | stores id of employee software license
+**employee_id**         | FK, INT | NOT NULL    | refers to the table employee
+**software_license_id** | FK, INT | NOT NULL    | refers to the table software_license
+**date_of_issue**       | DATE    | -           | stores info when license was issued
+**pick_up_date**        | DATE    | -           | stores info when license was picked up
```
Comment (aartiukh) [inline docs/diagram.md]: Rename  `pick_up_date` ->  `fisrt_usage_date`

File: docs/diagram.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table furniture consist of:
+
+var name     | type    | constraints | description
+------------ | ------- | ----------- | ----------------------------------
+**id**       | PK, INT | NOT NULL    | stores id of furniture
+**type_id**  | FK, INT | NOT NULL    | refers to the table furniture_type
+**name**     | TEXT    | -           | stores name of furniture
```
Comment (aartiukh) [inline docs/diagram.md]: Not empty

File: docs/diagram.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+**warranty** | INT     | -           | stores duration of warranty
```
Comment (aartiukh) [inline docs/diagram.md]: Specify units (months)
constraints `>=0`

File: docs/diagram.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+**cost**     | INT     | -           | stores cost of furniture
```
Comment (aartiukh) [inline docs/diagram.md]: Specify units (UAH)
constraints `>=0`

File: docs/diagram.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table furniture_type consist of:
+
+var name | type     | constraints | description
+-------- | -------- | ----------- | -------------------------------------------
+**id**   | PK, INT  | NOT NULL    | stores id of furniture types
+**type** | CHAR(64) | -           | stores info of different types of furniture
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+**date_of_issue** | DATE    | -           | stores info when furniture was issued
+
+## Table employee consist of:
+
+var name       | type        | constraints | description
+-------------- | ----------- | ----------- | ------------------------------------
+**id**         | PK, INT     | NOT NULL    | stores id of a employee
+**office_id**  | FK, INT     | NOT NULL    | refers to the table office
+**project_id** | FK, INT     | NOT NULL    | refers to the table project
+**name**       | CHAR(65)    | -           | stores the full name of the employee
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+**email**      | CHAR(65)    | -           | stores employee email
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+**phone**      | VARCHAR(15) | -           | stores employee phone number
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table equipment consist of:
+
+var name              | type          | constraints | description
+--------------------- | ------------- | ----------- | ------------------------------------------------
+**id**                | PK, INT       | NOT NULL    | stores id of a employee
+**equipment_type_id** | FK, INT       | NOT NULL    | refers to the table equipment_type
+**name**              | FK, CHAR(255) | NOT NULL    | stores the name of the equipment
+**warranty**          | INT           | -           | stores duration of warranty
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: >=0
specify units

File: docs/diagram.md [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+**cost**              | INT           | -           | stores equipment cost
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: >=0
specify units

File: docs/diagram.md [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+**status**            | CHAR(65)      | -           | stores information about the status of equipment
```
Comment (aartiukh) [inline docs/diagram.md]: list all possible values
constraints: Not empty

File: docs/diagram.md [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+**description**       | CHAR(255)     | -           | stores additional information about equipment
+**purhase_date**      | DATE          | -           | stores date of purchase
+**serial_number**     | DATE          | -           | stores serial number of equipment
+**phone**             | VARCHAR(15)   | -           | stores employee phone number
```
Comment (aartiukh) [inline docs/diagram.md]: :open_mouth:

File: docs/diagram.md [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table project consist of:
+
+var name | type     | constraints | description
+-------- | -------- | ----------- | ------------------------------
+**id**   | PK, INT  | NOT NULL    | stores id of a project
+**name** | CHAR(65) | -           | stores the name of the project
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 18]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table office consist of:
+
+var name | type      | constraints | description
+-------- | --------- | ----------- | -----------------------------
+**id**   | PK, INT   | NOT NULL    | stores id of a office
+**name** | CHAR(255) | -           | stores the name of the office
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 19]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table equipment_type consist of:
+
+var name | type      | constraints | description
+-------- | --------- | ----------- | -------------------------------------
+**id**   | PK, INT   | NOT NULL    | stores id of a equipment type
+**name** | CHAR(255) | -           | stores the name of the equipment type
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 20]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table equipment_part consist of:
+
+var name         | type      | constraints | description
+---------------- | --------- | ----------- | -------------------------------------
+**id**           | PK, INT   | NOT NULL    | stores id of a equipment part
+**equipment_id** | FK, INT   | NOT NULL    | refers to the table equipment
+**name**         | CHAR(255) | -           | stores the name of the equipment part
```
Comment (aartiukh) [inline docs/diagram.md]: constraints: Not empty

File: docs/diagram.md [hunk 21]
```diff
@@ -0,0 +1,122 @@
...
+
+## Table employee_equipment consist of:
+
+var name          | type     | constraints | description
+----------------- | -------- | ----------- | ----------------------------------------
+**id**            | PK, INT  | NOT NULL    | stores id of a employee_equipment
+**employee_id**   | FK, INT  | NOT NULL    | refers to the table employee
+**equipment_id**  | FK, INT  | NOT NULL    | refers to the table equipment
+**name**          | CHAR(65) | -           | stores the full name of the employee
```
Comment (aartiukh) [inline docs/diagram.md]: :open_mouth:

File: docs/diagram.md [hunk 22]
```diff
@@ -0,0 +1,122 @@
...
+**date_of_issue** | DATE     | -           | stores info when equipment was issued
+**pick_up_date**  | DATE     | -           | stores info when equipment was picked up
```
Comment (aartiukh) [inline docs/diagram.md]: rename

**Review by aartiukh — CHANGES_REQUESTED** [review]

File: docs/diagram.md [hunk 23]
```diff
@@ -0,0 +1,121 @@
+# Description of database structure
+
+![database.png](images/database.png)
+
+## Table software consist of:
+
+var name | type    | constraints | description
+-------- | ------- | ----------- | ----------------------
+**id**   | PK, INT | NOT NULL    | stores id of software
+**name** | TEXT    | NOT NULL           | store name of software
```
Comment (aartiukh) [inline docs/diagram.md]:
```suggestion
**name** | TEXT    | NOT NULL           | stores name of software
```

File: docs/diagram.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
+# Description of database structure
+
+![database.png](images/database.png)
+
+## Table software consist of:
+
+var name | type    | constraints | description
```
Comment (aartiukh) [inline docs/diagram.md:7]: Please fix indentations in the tables.

File: docs/diagram.md [hunk 25]
```diff
@@ -0,0 +1,121 @@
...
+**fisrt_usage_date**        | DATE    | NOT NULL           | stores info when license was picked up
+
+## Table furniture consist of:
+
+var name     | type    | constraints | description
+------------ | ------- | ----------- | ----------------------------------
+**id**       | PK, INT | NOT NULL    | stores id of furniture
+**type_id**  | FK, INT | NOT NULL    | refers to the table furniture_type
+**name**     | TEXT    | NOT NULL           | stores name of furniture
+**warranty** | INT     | NOT NULL, REAL(months)           | stores duration of warranty
```
Comment (aartiukh) [inline docs/diagram.md]:
```suggestion
**warranty** | INT     | NOT NULL, REAL (months)           | stores duration of warranty
```

**Review by aartiukh — DISMISSED** [review]

File: docs/diagram.md (hunk 24 above)
Comment (MykytaKyt) [reply docs/diagram.md:7]: fixed

**Review by aartiukh — DISMISSED** [review]
LGTM

**Comment by MykytaKyt** [comment]
Final fix in picture

**Review by OleksandrZhytnyk — DISMISSED** [review]

**Review by aartiukh — CHANGES_REQUESTED** [review]
Why two images files?
```
docs/images/database.png 
docs/images/diagram.png 
```
Remove unused one.

**Review by aartiukh — APPROVED** [review]

**Review by OleksandrZhytnyk — APPROVED** [review]

**Pull request merged**